app/src/main/res/values/strings.xml

```diff
@@ -191,4 +191,5 @@
   <string name="your_languages">Selected languages:</string>
   <string name="other_languages">Other languages:</string>
     <string name="no_items_msg">No items available</string>
+    <string name="Sharingmessage">The app is a lightweight piece of software reading bigger files stored on your device or SD card: once it is installed, you can select which additional content you would like to download (Wikipedia, Wiktionary, TED talks, etc.) and be ready for when your internet connexion is bad (or need to be in airplane mode)! Download Kiwix Now:https://play.google.com/store/apps/details?id=org.kiwix.kiwixmobile   Note: Kiwix is also available on regular computers (Windows, Mac, Linux).</string>
```

Comment:
This is too long and the name does not follow the rest of the messages. It is also indented wrong (along with the item above it).